README.md

```
@@ -2,133 +2,88 @@
+<br/>
 
-## 서비스 탄생 배경
+## 서비스 개요
 
-> 포게더는 '특별한 순간을 함께 모으는 사진 공유 서비스'에요.
+> 포게더는 '작가와 방문객에 연결되는 공간 서비스'에요.
 
-호스트는 스페이스를 생성해 생일파티, 졸업식, 결혼식 사진을 다수에게 받을 수 있어요.  
-게스트는 링크로 접속해 이미지를 업로드할 수 있어요.   
-스페이스에서 사진을 조회하고, 원하는 사진을 다운로드 받아요.
+시간과 공간이 부족해 전시 공간을 꾸미지 못하는 작가들의 문제에 주목했어요.   
```

markdownlint-cli2 (0.18.1)

Trailing spaces
Expected: 0 or 2; Actual: 3

(MD009, no-trailing-spaces)